google/cloud/spanner/value_test.cc

```diff
@@ -753,6 +754,21 @@ TEST(Value, ProtoConversionNumeric) {
   }
 }
 
+TEST(Value, ProtoConversionPgOid) {
+  for (auto const& x : std::vector<PgOid>{
+           PgOid("0"),
+           PgOid("200"),
+           PgOid("999"),
+       }) {
+    Value const v(x);
+    auto const p = spanner_internal::ToProto(v);
+    EXPECT_EQ(v, spanner_internal::FromProto(p.first, p.second));
+    EXPECT_EQ(p.first.code(), google::spanner::v1::TypeCode::INT64);
+    EXPECT_EQ(p.first.type_annotation(), google::spanner::v1::TypeAnnotationCode::PG_OID);
```

Comment:
Run the `checkers` build to update the formatting.

Reply:
Done